### ISSUE TITLE
Fatal Error on plugin (network) activation

### ISSUE THREAD

**Issue by ollyguv** [coinbase/coinbase-wordpress#1]
Plugin could not be activated because it triggered a fatal error.

Warning: require_once(/PATH/TO/public/wp-content/plugins/coinbase-wordpress-master/coinbase-php/lib/Coinbase.php): failed to open stream: No such file or directory in /PATH/TO/public/wp-content/plugins/coinbase-wordpress-master/coinbase.php on line 13 Fatal error: require_once(): Failed opening required '/PATH/TO/public/wp-content/plugins/coinbase-wordpress-master/coinbase-php/lib/Coinbase.php' (include_path='.:/usr/share/php:/usr/share/pear') in /PATH/TO/public/wp-content/plugins/coinbase-wordpress-master/coinbase.php on line 13

...

basically, lib/Coinbase.php is missing from wp-content/plugins/coinbase-wordpress-master/coinbase-php

Should I obtain that from elsewhere?

(I'm interested in developing this plugin for s2Member compatibility, BTW.)

Thank you.
